external/mmdetection/tests/test_ote_api.py

```diff
@@ -541,6 +541,18 @@ def test_training_yolox(self):
         self.end_to_end(
             osp.join('configs', 'custom-object-detection', 'cspdarknet_YOLOX'))
 
+    @e2e_pytest_api
+    def test_training_person_detection_0301(self):
+        self.end_to_end(osp.join('configs', 'person-detection', 'person-detection-0301'))
+
```

Comment:
I'm still not sure how we should add tests for object detection tasks. If we need to add it this way, why don't you add tests for the 020* person detectors?